DNN Platform/DotNetNuke.Web/InternalServices/ControlBarController.cs

```diff
@@ -586,6 +586,10 @@ private int DoAddExistingModule(int moduleId, int tabId, string paneName, int po
                             break;
                     }
                 }
+                else if (!ModulePermissionController.HasModuleAccess(SecurityAccessLevel.Edit, "MANAGE", moduleInfo))
```

Comment:
Do we need to do this check even if `remote` is true?  Should this be `if` instead of `else if`?

Reply:
I was wondering about that too. Also @zyhfish would it be possible to resubmit this against the release/9.8.0 branch, we would love to get this into the current 9.8.0 RC

Reply:
PR #4238 submitted to release branch, thx